### PR TITLE
Add all missing texts and translate them.

### PR DESCRIPTION
**Pull request by ffes**
As mentioned in #30 some text where still missing. This adds
all those texts and adds the texts that were added by #26 as well.

### CODE REVIEW

**Comment by mattock** [comment]
I had a quick look and saw no obvious issues. As @syzzer approved the general quality of the translations in https://github.com/OpenVPN/openvpn-gui/pull/30 already, I don't see a need for a separate linguistic review. Merging, thanks!

File: res/openvpn-gui-res-nl.rc [hunk 1]
```diff
@@ -198,6 +199,17 @@ BEGIN
...
+    IDS_ERR_CONFIG_NOT_AUTHORIZED "Het maken van deze verbinding (%s) vereist lidmaatschap van\n"\
+                                  "de ""%s""-groep. Neem contact op met de systeembeheerder.\n"
+    IDS_ERR_CONFIG_TRY_AUTHORIZE  "Het maken van deze verbinding (%s) vereist lidmaatschap van\n"\
+                                  "de ""%s""-groep.\n\n"\
+                                  "Wilt u uzelf toevoegen aan deze groep?\n"\
+                                  "Tijdens deze actie kan er om het administratorwachtwoord gevraagd worden."
+    IDS_NFO_CONFIG_AUTH_PENDING   "Het maken van deze verbinding (%s) vereist lidmaatschap van\n"\
+                                  "de ""%s""-groep.\n\n"\
+                                  "Vul het voorgaande authorizationscherm volledig in."
+    IDS_ERR_ADD_USER_TO_ADMIN_GROUP "Toevoegen van de grbruiker aan de ""%s""-groep is mislukt."
```
Comment (syzzer) [inline res/openvpn-gui-res-nl.rc]: Typo: grbruiker -> gebruiker

File: res/openvpn-gui-res-nl.rc [hunk 2]
```diff
@@ -198,6 +199,17 @@ BEGIN
...
+    IDS_NFO_NO_CONFIGS "Geen leesbare verbindingsprofielen (config files) gevonden."
+    IDS_ERR_CONFIG_NOT_AUTHORIZED "Het maken van deze verbinding (%s) vereist lidmaatschap van\n"\
+                                  "de ""%s""-groep. Neem contact op met de systeembeheerder.\n"
+    IDS_ERR_CONFIG_TRY_AUTHORIZE  "Het maken van deze verbinding (%s) vereist lidmaatschap van\n"\
+                                  "de ""%s""-groep.\n\n"\
+                                  "Wilt u uzelf toevoegen aan deze groep?\n"\
+                                  "Tijdens deze actie kan er om het administratorwachtwoord gevraagd worden."
+    IDS_NFO_CONFIG_AUTH_PENDING   "Het maken van deze verbinding (%s) vereist lidmaatschap van\n"\
+                                  "de ""%s""-groep.\n\n"\
+                                  "Vul het voorgaande authorizationscherm volledig in."
```
Comment (syzzer) [inline res/openvpn-gui-res-nl.rc]: authorizationscherm -> autorisatiescherm

**Comment by ffes** [comment]
@syzzer. Oops. You're right. I will open a new PR.

File: res/openvpn-gui-res-nl.rc (hunk 2 above)
Comment (mattock) [inline res/openvpn-gui-res-nl.rc]: Duh, I was a bit optimistic it seems :). Thanks for the review @syzzer.

**Pull request merged**